### DISCUSSION TITLE
Local netbox user login problem after migrating to 3.2

### DISCUSSION THREAD

**Discussion by pwoconnor** [netbox-community/netbox#11597] [Help Wanted!]
I have a user configured locally in netbox. Since the migration to 3.4 his account keeps getting kicked to inactive. I have deleted the user several times and after recreating the user all of his old info including password is added to back. Not quite sure where to go from here, I have spent several days searching online but have not come across anything that helps. Any ideas or recommendations would be appreciated.

**Comment by candlerb** [comment]
Are you using some other form of authentication (e.g. LDAP, RemoteUser or SocialAuth) in addition to local logins?

**Reply by pwoconnor** [reply]
Currently strictly local logins setup through the admin page.

**Reply by candlerb** [reply]
Then I don't know, especially "after recreating the user all of his old info including password is added to back". I suggest you try some SQL queries to confirm whether the user is actually being deleted, and if they are, at what point they are recreated, and whether they are recreated with the same id or a different one.

```
# sudo -u postgres psql netbox

select id,username,is_active,date_joined,last_login from auth_user;
```

**Reply by pwoconnor** [reply]
Yep I did verify that the user is dropped from the auth_user table in postgres when I delete them from the admin portal. When i recreate the user it populates and has a new user_id. Normally it does not boot to inactive right away, there is some time that passes. I havent been able to figure out what triggers but one day they will try to login and their password says invalid, if they try their old password it kicks them to inactive. 

Definitely a strange one...

**Reply by candlerb** [reply]
I think you want to do a full `select * ...`, wait for the problem to occur, and then repeat it. Check if the ID has changed again, check that the password hash has changed, and check the timestamp columns. Check Netbox's changelog just in case (although I'm not sure if user changes get logged).

I wonder if you might have some external integration which is pushing the old password back into the database - I can think of nowhere else it could come from.

**Reply by pwoconnor** [reply]
This issue just happened again. The user was marked inactive about 3 weeks ago. I deleted his account and recreated it. New user ID name was used (viewed in DB). Some how or another it is pulling info from the old account. It pulls the creation time from the first time the user was created. I have confirmed there are no integrations, we us local logins only.  

If I just check the boxes to make the user active, when the user tries to login it will immediately make them inactive. If we change the password and make them active again the account can be used for a few days before randomly being forced back to inactive...

**Reply by FrankPGH** [reply]
I just ran into this same thing. I have users I'm authenticating using LDAP and one of them reported that he was set to Inactive. I manually reset him back to Active in the Admin portal but it flipped him back to Inactive within a few minutes (or possibly when he tried to log in again). I'm not sure where or how this is logged to gather more information.

EDIT: I actually found my problem and it was essentially my fault, it just never appeared as a problem until one specific person tried to log in. I'm leaving the comment anyway in case it helps.

I'm using LDAP authentication with users assigned to an AD group. In ldap_config.py, I had AUTH_LDAP_REQUIRE_GROUP pointed to the correct AD group, but AUTH_LDAP_USER_FLAGS_BY_GROUP "is_active" was set to a similar (but not the same) group. My affected user was in one group but not the other. We just never noticed since this is the first time in two years he's tried to log into Netbox.
